src/components/EmptySelectionListContent.tsx

```diff
@@ -48,8 +48,8 @@ function EmptySelectionListContent({contentType}: EmptySelectionListContentProps
     );
 
     return (
-        <ScrollView>
-            <View style={[styles.flex1, styles.overflowHidden, styles.minHeight65]}>
+        <ScrollView contentContainerStyle={[styles.flex1, styles.justifyContentCenter]}>
+            <View style={[styles.overflowHidden, styles.minHeight65]}>
```

Comment:
```suggestion
        <ScrollView contentContainerStyle={[styles.flexGrow1]}>
            <View style={[styles.flex1, styles.overflowHidden, styles.minHeight65]}>
```

Let's only add `styles.flexGrow1` to the `ScrollView` bcoz `flex1` won't show the scrollbar for native if content exceeds.

https://github.com/user-attachments/assets/0bb883fd-e1a7-4be1-a2cb-4c5f53bbac07